flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/validation/DefaultValidator.java

```diff
@@ -605,9 +605,16 @@ public static Optional<String> validateAutoScalerFlinkConfiguration(
         return firstPresent(
                 validateNumber(flinkConfiguration, AutoScalerOptions.MAX_SCALE_DOWN_FACTOR, 0.0d),
                 validateNumber(flinkConfiguration, AutoScalerOptions.MAX_SCALE_UP_FACTOR, 0.0d),
-                validateNumber(flinkConfiguration, AutoScalerOptions.TARGET_UTILIZATION, 0.0d),
+                validateNumber(flinkConfiguration, AutoScalerOptions.UTILIZATION_TARGET, 0.0d),
                 validateNumber(
                         flinkConfiguration, AutoScalerOptions.TARGET_UTILIZATION_BOUNDARY, 0.0d),
+                validateNumber(flinkConfiguration, AutoScalerOptions.UTILIZATION_MAX, 0.0d, 1.0d),
+                validateNumber(flinkConfiguration, AutoScalerOptions.UTILIZATION_MIN, 0.0d, 1.0d),
```

Comment:
It seems we don't need a new `validateNumberOrder`, and following change is enough. 

```suggestion
                validateNumber(flinkConfiguration, AutoScalerOptions.UTILIZATION_MAX, flinkConfiguration.get(AutoScalerOptions.UTILIZATION_TARGET), 1.0d),
                validateNumber(flinkConfiguration, AutoScalerOptions.UTILIZATION_MIN, 0.0d, flinkConfiguration.get(AutoScalerOptions.UTILIZATION_TARGET)),
```

Reply:
makes sense